tech-committee-charter.md

```diff
@@ -0,0 +1,88 @@
+# OpenTelemetry Technical Committee (TC) Charter
+
+## Guiding Principle
+
+The OpenTelemetry project will operate transparently, collaboratively, ethically, and in alignment with the interests of OpenTelemetry technology end-users. Project proposals, timelines, and status must not merely be open, but also easily visible to and discoverable by outsiders.
+
+## Evolution of OpenTelemetry Governance
+
+Most large, complex open source communities have both a business and a technical governance model. Since its inception, the OpenTelemetry project strives to combine and align the OpenTracing and OpenCensus projects. OpenTelemetry has a project-level Governance Committee, a project-level Technical Committee, and is a sandbox project in the Cloud Native Computing Foundation, working for interoperability and engagement across the industry in conjunction with the CNCF's Board of Directors and Technical Oversight Committee.
+
+This OpenTelemetry Technical Committee Charter reflects the scope and expectations of the Technical Committee ("TC") with the maintainers, the consumers, and the ecosystem of the project. The charter will not be perfect, and so has a simple amendment process – a TC member proposes changes to be discussed and voted on by the full TC.
+
+## Establishment of the Technical Committee
+
+TC memberships are not time-limited. There is no maximum size of the TC. The size is expected to vary in order to ensure adequate coverage of important areas of expertise, balanced with the ability to make decisions efficiently. The TC must have at least four members and is recommended to have an odd number of members for tie-breaking purposes.
```

Comment:
Pasting discussion from the monthly governance meeting: we should consider making these time limited

Reply:
are there arguments in favor fo time limited?  The downside is that then TC would need to have formal election process similar to GC

Reply:
The governance committee met today and discussed this topic (among others).

Briefly, we believe that there is a tradeoff between the stability and velocity of the project and robust checks and balances on TC members (including term limits). **Given the current project stage** (read: "very early and highly dynamic / chaotic"), we believe that OpenTelemetry's goals will be best-served by leaving term limits out of the TC charter *for now*, but we also recognize that this may very well shift in the future as the technical landscape around OpenTelemetry stabilizes.

As such, our resolution is to leave this section as-is for now (i.e., "no term limits"), **but** to intentionally reconsider this topic no later than **July 2020**; if the balance has shifted towards "checks and balances" and away from "stability" at that time, we may very well introduce term limits, a formal TC election process, and so forth.